lib/statsd/instrument/datagram_builder.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+class StatsD::Instrument::DatagramBuilder
+  unless Regexp.method_defined?(:match?) # for ruby 2.3
+    module RubyBackports
+      refine Regexp do
+        def match?(str)
+          (self =~ str) != nil
```

Comment:
total nitpick, but `match(str) != nil` would trigger my OCD less, and would avoid setting the useless `$` magic variables.

Not a huge diff though.

Reply:
Fixed!